src/eei.h

```diff
@@ -76,6 +76,8 @@ struct EthereumInterface : ShellExternalInterface {
 
   // EEI methods
 
+  uint32_t eeiGetReturnDataSize();
```

Comment:
Refer to #148 for the ordering of these function declarations. I think from a maintainability perspective it makes sense to have a more logical ordering of the methods (the design doc needs to also be re-ordered).

Reply:
Yeah, would need to apply that to both design, hera, ewasm-rust-api, as-ewasm-api.

But perhaps not on this PR.